documentation/app/templates/index.hbs

```diff
@@ -11,6 +11,7 @@
   <T.Panel>
     <div class="doc-Page">
       <div class="doc-Page-content">
+        <Hds::Tag @text="Test tag" />
```

Comment:
do we want to remove this tag?

Reply:
oh holy smokes I though I dropped that commit I must have accidentally squashed it. Thanks!